test/OrchardCore.Tests/Shell/ShellHostTests.cs

```diff
@@ -10,54 +9,32 @@ namespace OrchardCore.Tests.Shell;
 
 public class ShellHostTests : SiteContext
 {
-    private readonly IShellHost _shellHost;
+    public static IShellHost ShellHost { get; }
```

Comment:
@jtkech why you mark it as static? @CrestApps already did that at the beginning but I asked him to change it ;)

Reply:
Yes I saw this, because Site is a static thing so all can be static